src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -125,6 +126,10 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
                     name: PSSerializeJSONLongEnumAsNumber,
                     description: "Serialize enums based on long or ulong as an numeric value rather than the string representation when using ConvertTo-Json."
+                ),
+                new ExperimentalFeature(
+                    name: PSDnsNameListCertificateParser,
+                    description: "Parses Common Name when adding to DnsNameList for Subject Distinguished names using X500DistinguishedName.EnumerateRelativeDistinguishedNames."
```

Comment:
Please reword the description so that users understand that we use new strong algorithm to parse Subject Distinguished name to get DNS names.

Reply:
Reworded in c23995478 to `Parses Subject Distinguished name from Common Name using strong algorithm to get DNS names.`.

Reply:
Also maybe experimental feature should be renamed to something more along these lines? Hard to get a better name which includes "Subject Distinguished name" without sounding too long.

Reply:
You can try.
And please see @doctordns comment below.